src/main/flight/gps_rescue.c

```diff
@@ -72,7 +72,8 @@ PG_RESET_TEMPLATE(gpsRescueConfig_t, gpsRescueConfig,
     .throttleMax = 1600,
     .throttleHover = 1280,
     .sanityChecks = RESCUE_SANITY_ON,
-    .minSats = 8
+    .minSats = 8,
+    .minRescueDTH = 100
```

Comment:
Correct camelCase is `minRescueDth`.

Reply:
I can change this, but I really do think capitalized is the correct way to camelcase with short acronyms and abbreviations.  DTH is distance to home, and while this style is not entirely consistent between BF code base, I think more often than not <= 3 character abbreviations are capitalized.  (e.g sbufWriteU16(dst, GPS_distanceToHome);

Reply:
CamelCase is generally done by only capitalising the first letter of an abbreaviation. One of the reasons for this is that it allows consistent capitalisation in cases where the abbreviation is the first part. If we were to follow your suggestion, and then later on added support for different units for the DTH distance, this would have to go into `dTHUnits`, which is ugly and confusing.